tests/integration-tests/tests/create/test_create.py

```diff
@@ -117,6 +119,29 @@ def test_create_imds_secured(
     assert_cluster_imds_v2_requirement_status(region, cluster, status)
 
 
+# FIXME: Move in tests_update after update policy is changed
+@pytest.mark.usefixtures("instance", "scheduler")
+def test_create_disable_sudo_access_for_default_user(
+    region, os, scheduler, pcluster_config_reader, clusters_factory, architecture, scheduler_commands_factory
+):
+    """
+    Verify that the cluster removes the Sudo access for Default user
+    in all the nodes of the Cluster if the DisableSudoAccessForDefaultUser is enabled.
+    """
+    disable_sudo_access_default_user = True
+    cluster_config = pcluster_config_reader(disable_sudo_access_default_user=disable_sudo_access_default_user)
+    cluster = clusters_factory(cluster_config, raise_on_error=True)
+
+    logging.info("Checking default user has disabled sudo access after cluster creation")
+    assert_head_node_is_running(region, cluster)
+    for node_type in NodeType:
+        if scheduler == "awsbatch" and node_type.value == "LoginNodes":
```

Comment:
Shouldn't it be singular: LoginNode?

Reply:
Will not check for another Hardcoded Value